src/multi.jl

```diff
@@ -0,0 +1,77 @@
+# Univariate
+
+const FillVectorOfUnivariate{
+    S <: ValueSupport,
+    T <: UnivariateDistribution{S},
+    Tdists <: Fill{T, 1},
+} = VectorOfUnivariate{S, T, Tdists}
+
+function FillDist(dist::UnivariateDistribution, N::Int)
+    return Product(Fill(dist, N))
```

Comment:
Again I think it would be preferable to not call `Product` but define
```suggestion
    return product_distribution(Fill(dist, N))
```